server/src/main/java/io/crate/execution/dsl/projection/builder/InputColumns.java

```diff
@@ -262,6 +263,15 @@ public Symbol visitReference(Reference ref, SourceSymbols sourceSymbols) {
         return inputColumn;
     }
 
+    @Override
+    public Symbol visitVoidReference(VoidReference ref, SourceSymbols sourceSymbols) {
+        var symbol = lookupValueByColumn(sourceSymbols.inputs, ref.column());
+        if (symbol != null) {
+            return symbol;
+        } else {
+            return visitReference(ref, sourceSymbols);
+        }
+    }
```

Comment:
This prevents `UnsupportedFeatureException[Can't handle Symbol [VoidReference: obj_dynamic['u']]]` when `SET error_on_unknown_object_key = 'false'`.